lib/explorer/series.ex

```diff
@@ -5355,12 +5359,46 @@ defmodule Explorer.Series do
   """
   @doc type: :string_wise
   @spec contains(Series.t(), String.t()) :: Series.t()
-  def contains(%Series{dtype: :string} = series, pattern)
-      when K.is_binary(pattern),
-      do: apply_series(series, :contains, [pattern])
+  def contains(%Series{dtype: :string} = series, substring)
+      when K.is_binary(substring),
+      do: apply_series(series, :contains, [substring])
 
   def contains(%Series{dtype: dtype}, _), do: dtype_error("contains/2", dtype, [:string])
 
+  @doc """
+  Detects whether a string matches a pattern.
+
+  > ### Notice {: .warning}
+  >
+  > This function matches against a regular expression. It does not expect an Elixir regex, but
+  > a escaped string - you can use the `~S` sigil for escaping - that follows the [`regex`](https://docs.rs/regex/latest/regex/)
+  > Rust crate rules. This is because our backend, Polars, expects that format.
+  >
+  > To match literal strings, you can use `contains/2`.
```

Comment:
```suggestion
  > This function matches against a regular expression. It does not expect an Elixir regex,
  > but a escaped string and you can use the `~S` sigil for escaping it. Since each Explorer
  > backend may have its own regular expression rules, you must consult their underlying
  > engine. For the default backend (Polars), the rules are outlined in the Rust create named
  > [`regex`](https://docs.rs/regex/latest/regex/).
  >
  > To match literal strings, you can use `contains/2`.
```

Reply:
Please mirror it below and ship it :D